pkg/ingress/model_builder.go

```diff
@@ -15,18 +21,62 @@ type ModelBuilder interface {
 	Build(ctx context.Context, ingGroup Group) (core.Stack, *elbv2model.LoadBalancer, error)
 }
 
+// NewDefaultModelBuilder constructs new defaultModelBuilder.
+func NewDefaultModelBuilder(k8sClient client.Client, eventRecorder record.EventRecorder, ec2Client services.EC2, vpcID string, clusterName string,
+	annotationParser annotations.Parser, subnetsResolver networking.SubnetsResolver,
+	authConfigBuilder AuthConfigBuilder, enhancedBackendBuilder EnhancedBackendBuilder) *defaultModelBuilder {
+	return &defaultModelBuilder{
+		k8sClient:              k8sClient,
+		eventRecorder:          eventRecorder,
+		ec2Client:              ec2Client,
+		vpcID:                  vpcID,
+		clusterName:            clusterName,
+		annotationParser:       annotationParser,
+		subnetsResolver:        subnetsResolver,
+		authConfigBuilder:      authConfigBuilder,
+		enhancedBackendBuilder: enhancedBackendBuilder,
+
+		defaultIPAddressType:                      elbv2model.IPAddressTypeIPV4,
+		defaultScheme:                             elbv2model.LoadBalancerSchemeInternal,
+		defaultSSLPolicy:                          "ELBSecurityPolicy-2016-08",
```

Comment:
Should we not leave it blank and let AWS choose the default?

Reply:
it's for compatible with old ingress controller. 
we can change this behavior in later version :D